lib/sinon/util/core/iterable-to-string.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+var typeOf = require("../../typeOf");
+
+module.exports = function iterableToString(obj) {
```

Comment:
Name anonymous function to aid debugging (ref [previous review](https://github.com/sinonjs/sinon/pull/1215#discussion_r94131858)).